resources/user.py

```diff
@@ -44,17 +44,11 @@ def get(self, user_id):
         user_info = user.json()
 
         if user.role == RoleEnum.PROPERTY_MANAGER:
-            user_info["properties"], tenant_list = zip(
-                *(
-                    (p.json(), p.tenants)
-                    for p in PropertyModel.find_by_manager(user_id)
-                    if p
-                )
-            )
+            properties_list = PropertyModel.find_by_manager(user_id)
+            user_info["properties"] = [p.json() for p in properties_list]
 
-            tenant_IDs = [tenant.id for sublist in tenant_list for tenant in sublist]
-            tenants_list = [TenantModel.find_by_id(t) for t in set(tenant_IDs)]
-            user_info["tenants"] = [t.json() for t in tenants_list if t]
+            tenants_json_2D = [p.tenants() for p in properties_list]
+            user_info["tenants"] = list(chain.from_iterable(tenants_json_2D))
```

Comment:
Much cleaner solution. Props to @DaveTrost!

Reply:
Since this method is being modified, perhaps it can be refactored and cleaned up a bit too. Update it to use the `find` method instead of `find_by_id`.

Also this serialization logic doesn't belong here in the endpoint. Perhaps we can move it to the `json` method. That way we don't have to duplicate this wherever we want to serialize a property manager.

Reply:
This can be removed now right?